src/plugins/intel_gpu/src/plugin/compiled_model.cpp

```diff
@@ -169,7 +169,7 @@ CompiledModel::CompiledModel(cldnn::BinaryInputBuffer ib,
     auto pos = ib.tellg();
     for (uint16_t n = 0; n < m_config.get_property(ov::num_streams); n++) {
         ib.seekg(pos);
-        auto graph = std::make_shared<Graph>(ib, context, m_config, 0);
+        auto graph = std::make_shared<Graph>(ib, context, m_config, n);
```

Comment:
could you implement a test for this or check why it was not captured from existing test?

Reply:
I updated the auto batch functional tests to include a multi-stream config. Thank you.